### PR TITLE
Updated Droplr to 4.6.4 (Build 91)

### PR DESCRIPTION
**Pull request by sbibauw**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Previous link (http://droplr.com/homebrew) links currently to an error

### CODE REVIEW

File: Casks/droplr.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'droplr' do
-  version :latest
-  sha256 :no_check
+  version '4.6.4'
+  sha256 '41064cec1922fa76b15e940a747e3d12e720588fa3ab9a527b38ba3a683c531b'
 
-  url 'https://droplr.com/homebrew'
+  url 'https://s3.amazonaws.com/files.droplr.com/apps/mac/Droplr+91.zip'
```
Comment (vitorgalvao) [inline Casks/droplr.rb]: This will not pass `style`. It will need a comment.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/droplr.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/droplr.rb]: Where did you get this exact URL? I’m getting a different one (that also fails, like the previous one).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/droplr.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/droplr.rb]: Run the `style` command, and you’ll see the problem. You need to add a `# s3.amazonaws.com/files.droplr.com was verified as official when first introduced to the cask` comment above `url`.

File: Casks/droplr.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 cask 'droplr' do
-  version :latest
-  sha256 :no_check
+  version '4.6.4'
```
Comment (vitorgalvao) [inline Casks/droplr.rb]: Make it `4.6.4,91`.

File: Casks/droplr.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/droplr.rb]: And here use `url "https://s3.amazonaws.com/files.droplr.com/apps/mac/Droplr+#{version.after_comma}.zip"`

**Comment by sbibauw** [comment]
I have applied all the recommendations:
```
version "4.6.4,91"
sha256 "41064cec1922fa76b15e940a747e3d12e720588fa3ab9a527b38ba3a683c531b"

# files.droplr.com.s3.amazonaws.com was verified as official when first introduced to the cask
url "http://files.droplr.com.s3.amazonaws.com/apps/mac/Droplr+#{version.after_comma}.zip"
```
but still get a `[BUG] Segmentation fault` error on `brew cask style --fix`

Anyway, I have contacted Droplr to have them correct the `https://droplr.com/homebrew` link. 

What I couldn't find is Guidelines about whether `version :latest` + `sha256 :no_check` are preferred to a specific version number (better for updating, but needs more maintenance), or a specific version number (with/without a flexible URL scheme) are preferred...

File: Casks/droplr.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/droplr.rb]: There is somewhere where we mention we prefer versions over `:latest` (users overwhelmingly voiced this preference).

Did you push your changes? It still looks the same.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by sbibauw** [comment]
`brew cask style --fix` still fails ("[BUG] Segmentation fault")

**Comment by vitorgalvao** [comment]
> `brew cask style --fix` still fails ("[BUG] Segmentation fault")

That’s an issue to report [on the core](https://github.com/Homebrew/brew). There was a similar issue a while ago, so please search for that first.

**Pull request merged**